src/OfficeToolPlus/Dictionaries/Languages/de-de.xaml

```diff
@@ -128,8 +128,8 @@
     <sys:String x:Key="Deploy_Architecture">Architektur</sys:String>
     <sys:String x:Key="Deploy_Architecture.x86">32 Bit</sys:String>
     <sys:String x:Key="Deploy_Architecture.x64">64 Bit</sys:String>
-    <sys:String x:Key="Deploy_Architecture.ARM">ARM (32-bit compatible)</sys:String>
-    <sys:String x:Key="Deploy_Architecture.ARM64">ARM64 (64-bit compatible)</sys:String>
+    <sys:String x:Key="Deploy_Architecture.ARM">ARM (32-Bit-kompatibel)</sys:String>
+    <sys:String x:Key="Deploy_Architecture.ARM64">ARM64 (64-Bit-kompatibel)</sys:String>
```

Comment:
In the previous line, it is `64 Bit`, not `64-Bit`. Is it correct?

Reply:
Yes, `64-Bit-kompatibel` is correct. What previous line do you mean?